src/App.js

```diff
@@ -1,13 +1,61 @@
 import React from 'react';
-
+import { GoodsList } from './components/index';
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRed } from './api/goods';
+
+class App extends React.Component {
+  state = {
+    goods: [],
+  }
+
+  visibleAll = () => {
+    getAll()
+      .then((goods) => {
+        this.setState({ goods });
+      });
+  }
+
+  visibleFive = () => {
+    get5First()
+      .then((goods) => {
+        this.setState({ goods });
+      });
+  }
+
+  visibleRedGoods = () => {
+    getRed()
+      .then((goods) => {
+        this.setState({ goods });
+      });
+  }
```

Comment:
read Olya's comment, you can indeed create a universal event handler which receives a callback